scene/2d/navigation_region_2d.cpp

```diff
@@ -165,6 +165,7 @@ void NavigationRegion2D::_notification(int p_what) {
 
 		case NOTIFICATION_INTERNAL_PHYSICS_PROCESS: {
 			set_physics_process_internal(false);
+			_region_update_transform();
```

Comment:
The way internal physics process is used here as a one shot seems like a convoluted way to do a deferred call. I assume call deferred wouldn't be enough for this method in `NOTOFICATION_TRANSFORM_CHANGED`?